docs/tsconfig.json

```diff
@@ -1,5 +1,5 @@
 {
-  "extends": "@docusaurus/tsconfig/tsconfig.json",
+  "extends": "@docusaurus/tsconfig",
```

Comment:
This didn't match Docusaurus's documentation before. I think it was failing to resolve the parent config until I fixed this, because TypeScript was complaining about uses of `import.meta` in CommonJS modules, but this config sets the module type to `esnext`.

Reply:
Good catch, I am surprised I didn't run into any issues in #2109 related to that.